src/commands/clone-backend-status.ts

```diff
@@ -0,0 +1,52 @@
+import {BaseCommand} from '../lib'
+import {flags} from '@oclif/command'
+
+const RESTORE_QUERY = `
+query($restoreId: Int!) {
+  restoreStatus(restoreId: $restoreId) {
+    response {status errors}
+  }
+}
+`
+
+export default class CloneBackendStatus extends BaseCommand {
+    static description = 'Retrieve the backend cloning status using clone key'
+
+    static hidden = true;
+
+    static examples = [
+      '$ slash-graphql clone-backend-status -e https://frozen-mango.cloud.dgraph.io/graphql -t <apiToken> --key <key>',
+    ]
+
+    static flags = {
+      ...BaseCommand.commonFlags,
+      ...BaseCommand.endpointFlags,
+      key: flags.integer({char: 'k', description: 'clone key received from clone operation', required: true}),
+    }
+
+    async run() {
+      const opts = this.parse(CloneBackendStatus)
+      const backend = await this.backendFromOpts(opts)
+
+      const {data, errors} = await backend.slashAdminQuery<{updateGQLSchema: {gqlSchema: {schema: string}}}>(RESTORE_QUERY, {
```

Comment:
these types are wrong right? it should be slashAdminQuery<status>. Is this passing typescript as it is somehow?